app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/SessionDetailsFragment.kt

```diff
@@ -161,7 +161,12 @@ class SessionDetailsFragment : Fragment() {
             CalendarSharing(requireContext()).addToCalendar(session)
         }
         viewModel.setAlarm.observe(viewLifecycleOwner) {
-            AlarmTimePickerFragment.show(this, SESSION_DETAILS_FRAGMENT_REQUEST_CODE)
+            AlarmTimePickerFragment.show(this, SESSION_DETAILS_FRAGMENT_REQUEST_KEY) { requestKey, result ->
+                if (requestKey == SESSION_DETAILS_FRAGMENT_REQUEST_KEY) {
+                    val alarmTimesIndex = result.getInt(AlarmTimePickerFragment.ALARM_TIMES_INDEX_BUNDLE_KEY)
```

Comment:
I'm not 100% sure, but maybe you want to add a similar 0-check here to check whether the value really exists? (like you did above for the string as well, and 0 seems to be the fallback value here - https://developer.android.com/reference/android/os/BaseBundle#getInt(java.lang.String) )

Reply:
:+1: Thx. Fixed in d868cad1337cbaec8270587c103ea44e360fbf22.

Reply:
Sorry, but have you marked the wrong commit? I can't see any fix there.

Reply:
Now I see, thanks :)